docs/Producing.md

```diff
@@ -27,7 +27,25 @@ The log about produced messages is printed in the `Kafka Producer Log` Output vi
 
 Record content can be randomized by injecting mustache-like placeholders of [faker.js properties](https://github.com/Marak/faker.js#api-methods), like ``{{name.lastName}}`` or ``{{random.number}}``. Some randomized properties can be localized via the `kafka.producers.fakerjs.locale` setting.
 

```

Comment:
The same seed is used for randomizing the key and the value, so the same randomized content can be injected in both places.

Reply:
fixed